thenewboston_node/business_logic/models/transaction.py

```diff
@@ -15,7 +15,7 @@
 class Transaction:
     recipient: str
     amount: int
-    fee: Optional[bool] = None
+    fee: Optional[bool] = False
```

Comment:
no, it must be None, so we do have to store this key on serialization

Reply:
also add a comment about, so other developers will not try make the same change as you did

Reply:
1. Why just not omit `fee` during serialization if `fee = False`. It's not serialized in the current implementation.
2. What's the point in having nullable boolean value? `None` value is the equivalent of `False`? Am I wrong?

Reply:
1. We have general removal of `None` keys here see `override_to_dict()` methods over the code (although False is checked for `fee` as well). Also `False` as default contradicts `Optional` typing hint.
2. `None` means value is not set, so we are free to set whatever we consider a default (so hypothetically if later we decide default to be True it will be consistent)

Reply:
Fixed https://github.com/thenewboston-developers/thenewboston-node/pull/83/files#diff-482f9548c1f94aeefa4b8b0fdf887b7d080c70c4ab50e0a7f6f0444f914fc031R18